CONTRIBUTING.md

```diff
@@ -115,15 +115,26 @@ From inside of VSCode, open the `launch.json` file and in the _Launch Integratio
     * Create a new commit with a message the same as the branch name.
     * Create a PR for this branch.
     * Wait for the PR to be merged into `main`
-1. Trigger a release build on Actions by adding a new tag on branch `main` named after the release, as above. Note that when you push to upstream, you will need to fully qualify the ref. A command like this will work:
+1. Switch to `main` and add a new tag on the `main` branch with your new version (named after the release), e.g.
+    ```bash
+    git checkout main
+    git tag v1.3.6
+    ```
 
+   If you've accidentally created a badly named tag, you can delete it via 
     ```bash
-    git push upstream refs/tags/v1.3.6
+    git tag -d badly-named-tag
     ```
+1. Push the new tag upstream:
+
+    ```bash
+    git push origin v1.3.6
```

Comment:
Should we still use `refs/tags/v1.3.6` here? I think we originally used this fully-qualified ref in case there was a branch or something that was also called `v1.3.6`? 🤔 :thinking-about-git-intensifies:

Also minor: The example previously used `upstream` instead of `origin`. I personally use `upstream` for this repo and `origin` for my fork, so `upstream` was correct, but this may be different for others! 🍴  🆙

Reply:
Ah yes, using a fork complicates things. Thanks!

I've added both options as I don't think everyone works with a fork (I don't use one specifically to avoid the overcomplication).